src/index.html

```diff
@@ -10,5 +10,159 @@
   <body>
     <div id="main"></div>
     <script src="./js/index.js"></script>
+    <span><nav>HOME</nav></span>
+    <span><button>DOWNLOAD MY FULL RESUME</button></span>
+    <div><h1>John Roberts</h1></div>
+    <div><p>BUSINESS EXPERT</p></div>
+    <div>
+      <img src="https://static.wixstatic.com/media/86f442ff0674441faadc2994b1b33e77.jpg/v1/crop/x_130,y_160,w_1320,h_1311/fill/w_248,h_244,al_c,q_80,usm_0.66_1.00_0.01/86f442ff0674441faadc2994b1b33e77.jpg"
+           alt="John Roverts"
+      />
+    </div>
+    <span><button>DOWNLOAD MY FULL RESUME</button></span>
+    <div><h2>ABOUT ME</h2></div>
+    <span>
+      <button><img src="LinkedIn.jpg" alt="LinkedIn" /></button>
+    </span>
+    <span>
+      <button><img src="Twitter.jpg" alt="Twitter" /></button>
+    </span>
+    <span>
+      <button><img src="Facebook.jpg" alt="Facebook" /></button>
+    </span>
+    <span>
+      <p>
+        I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” or double click me to add your own content and make changes to the font. <br />
+        Feel free to drag and drop me anywhere you like on your page. I’m a great place for you to tell a story and let your users know a little more about you.
+      </p>
+    </span>
+    <span>
+      <p>
+          I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” or double click me to add your own content and make changes to the font. Feel free to drag and drop me anywhere you like on your page. I’m a great place for you to tell a story and let your users know a little more about you.
+      </p>
+    </span>
+    <div><h2>EXPERIENCE</h2></div>
+    <span>
+      <ul>
+        <li>
+          <p>
+            Budget Director<br />
+            @Wix.com<br />
+            2014 - Current<br />
+            <br />
+            This is a description of your work experience. Concisely describe your previous position in a way that’s easy for readers to scan quickly.
+          </p>
+        </li>
+        <li>
+          <p>
+              Business Administration<br />
+              @BizzNizz.net<br />
+              2010 - 2013<br />
+              <br />
+              This is a description of your work experience. Concisely describe your previous position in a way that’s easy for readers to scan quickly.
+          </p>
+        </li>
+        <li>
+          <p>
+              Finance Director<br />
+              @Computers & More LTD.<br />
+              2008 - 2012<br />
+              <br />
+              This is a description of your work experience. Concisely describe your previous position in a way that’s easy for readers to scan quickly.
+          </p>
+        </li>
+      </ul>
+    </span>
+    <div><h2>EDUCATION</h2></div>
```

Comment:
Don't overuse div as h2 is already a block tag and there is no need for it to be wrapped with another block tag like this.